components/browser/session/src/main/java/mozilla/components/browser/session/Session.kt

```diff
@@ -49,6 +50,7 @@ class Session(
         fun onFindResult(session: Session, result: FindResult) = Unit
         fun onDesktopModeChanged(session: Session, enabled: Boolean) = Unit
         fun onFullScreenChanged(session: Session, enabled: Boolean) = Unit
+        fun onThumbNailChanged(session: Session, bitmap: Bitmap?) = Unit
```

Comment:
let's rename this to `onThumbnailChanged` (lower case n). Thumbnail is one word.